composer.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "open-orchestra/open-orchestra-function-bundle",
+    "description": "A workflow function bundle for the OpenOrchestra Project",
+    "license": "Apache-2.0",
+    "authors": [
+        {
+            "name": "OpenOrchestra community",
+            "homepage": "https://github.com/open-orchestra/open-orchestra-function-bundle/graphs/contributors"
+        }
+    ],
+    "minimum-stability": "dev",
+    "require": {
+        "open-orchestra/open-orchestra-model-interface": "*",
+        "doctrine/data-fixtures":"1.1.1",
+        "doctrine/mongodb-odm": "1.0.0-BETA12",
+        "doctrine/mongodb-odm-bundle" : "3.0.0-BETA6",
+        "stof/doctrine-extensions-bundle": "1.1.0"
```

Comment:
do you really need all the stof/doctrine-extensions and not only the gedmo one ?